docs/release-notes/self-hosted/self-hosted-v10.0.0.md

```diff
@@ -22,7 +22,7 @@ Follow the steps below to upgrade to Codacy Self-hosted v10.0.0:
 
 1.  Update your Codacy command-line tools to the following versions:
 
-    -   [Codacy Analysis CLI 7.6.6](https://github.com/codacy/codacy-analysis-cli/releases/tag/7.6.6)
+    -   [Codacy Analysis CLI 7.7.4](https://github.com/codacy/codacy-analysis-cli/releases/tag/7.7.4)
```

Comment:
@lolgab can you please confirm that this version of the Codacy Analysis CLI (that you [tagged as `self-hosted-10.0.0`](https://github.com/codacy/codacy-analysis-cli/releases/tag/self-hosted-10.0.0)) is indeed compatible with Codacy Self-hosted 10.0.0?